Tests/LegacyUnit/SchedulerTasks/MailNotifierTest.php

```diff
@@ -1226,12 +1243,18 @@ public function sendRemindersToOrganizersForEventWithAttendancesAndAttachCsvFile
             ]
         );
 
-        $this->subject->sendEventTakesPlaceReminders();
+        $this->email = $this->getMockBuilder(MailMessage::class)
```

Comment:
Instead, please either modify the trait (and add test that `attach` is mocked), or call `getChildren` on the email to get the attachments (which I would recommedt).